src/plugins/validation/2and3/semantic-validators/items-required-for-array-objects.js

```diff
@@ -10,8 +10,24 @@
 
 const { walk } = require('../../../utils');
 const MessageCarrier = require('../../../utils/messageCarrier');
+const at = require('lodash/at');
 
-module.exports.validate = function({ jsSpec }) {
+const reduceObj = function(jsSpec, obj) {
+  if (obj['$ref']) {
+    var objPath = obj['$ref'].split('/');
```

Comment:
I meant to comment on this earlier, but we shouldn't use `var` anywhere. These should all be `const`.